packages/react-components/theme-designer/src/components/Export/ExportPanel.tsx

```diff
@@ -46,26 +46,85 @@ const useStyles = makeStyles({
     height: '100%',
     boxSizing: 'border-box',
   },
+  panelContainer: {
+    zIndex: 100,
+    position: 'absolute',
+    top: '0px',
+    right: '0px',
+    width: '400px',
+    border: `1px solid ${tokens.colorNeutralStroke1}`,
+    borderRadius: tokens.borderRadiusXLarge,
+    backgroundColor: tokens.colorNeutralBackground1,
+    boxShadow: tokens.shadow64,
+    padding: '16px',
+  }
 });
 
+const ExportPanelHeader = ({ onClose }: { onClose: () => void }) => {
+  const styles = useStyles();
```

Comment:
nit, nyc: I like to add the component name to the style function. This helps us share them clearly later on.

So this one would be useExportPanelHeaderStyles.

This is a habbit from writing shared library code that often ends up benefiting product vertical code.